cedar-policy-validator/src/cedar_schema/to_json_schema.rs

```diff
@@ -172,38 +187,46 @@ impl TryFrom<Namespace> for json_schema::NamespaceDefinition<RawName> {
         // Convert action decls, collecting all errors
         let actions = collect_all_errors(action.into_iter().map(convert_action_decl))?
             .flatten()
+            .map(|(key, value)| (key, value))
```

Comment:
what does this do?

Reply:
noticed the same thing in json_schema.rs lines 330 and 334

Reply:
Good catch. Fixed.